client/signup/main.jsx

```diff
@@ -329,6 +331,11 @@ const Signup = React.createClass( {
 			null;
 	},
 
+	pageTitle() {
+		const accountFlowName = 'account';
+		return this.props.flowName === accountFlowName ? translate( 'Create an account' ) : translate( 'Create a site' );
```

Comment:
Thanks. I checked client/signup/config/flows.js and account is the only one that is different, the rest are variations on creating a site